wiki/Beatmap/Pattern/Mania/Long_Notes/en.md

```diff
@@ -0,0 +1,29 @@
+---
+stub: true
+tags:
+  - jacks
+---
+
+# Long Notes
```

Comment:
"hold note" is the preferred terminology (but it's fine to mention long notes in parentheses in the lead paragraph)

Reply:
Is it really? I feel like it would be confusing otherwise to not standardize it with the mania RC. Can make the change but it would make sense to edit the RC to use that term as well